### ISSUE TITLE
how do you determine the label of reward model?

### ISSUE THREAD

**Issue by eyuansu62** [RUCAIBox/Slow_Thinking_with_LLMs#9]
In the first paper, the score training method is used for reward model. I am curious how do you determine the label of reward model. 
And how do you compute the loss of reward model, for example, cross_entropy(label (0.8), prob_of_yes(0.8))?

**Comment by Timothy023** [comment]
We utilize the ground truth answer to determine the label of the reward model.

For example, if a response is correct, the label of this response is 1.0 and the loss of reward model on this response is cross_entropy(label(1.0), prob_of_yes(1.0)).

**Comment by eyuansu62** [comment]
Thanks
